pkg/build/api/v1beta3/types.go

```diff
@@ -382,6 +386,21 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// Shell is an action that executes a shell script in a container
+	// running the build output image.
+	Shell string `json:"shell,omitempty" description:"shell script to be executed in a container running the build output image"`
```

Comment:
These are args to be passed to the shell script, so that you can get correct escaping, i.e.:

```
args = ["test", "other", "with spaces"]
script = "echo $0 $1 $2"

/bin/bash -c "echo $0 $1 $2" -- "test" "other" "with spaces"
```

For any user who has bash, I think they'd expect bash.  Some environments definitely symlink one to the other.  Doesn't docker cmd require bash?